modals/ideas.js

```diff
@@ -0,0 +1,13 @@
+const { getIdeas, setIdeas } = require('./utils/storage.js');
+
+function deleteIdea(ideaId) {
+    let ideas = getIdeas();
+
+    let ideaToDelete = ideas.filter(idea => ideaId !== idea.id);
```

Comment:
I would rename `ideaToDelete` to `updatedIdeas`

Reply:
Done.